ticdc/ticdc-sink-to-kafka.md

```diff
@@ -249,10 +249,18 @@ Sample configuration:
 
 ```toml
 [scheduler]
-# Set it to "true" to enable this feature.
+# The default value is "false". Set it to "true" to enable this feature.
 enable-table-across-nodes = true
 # When you enable this feature, it only takes effect for tables with the number of regions greater than the `region-threshold` value.
 region-threshold = 100000
+# When you enable this feature, it takes effect for tables with the number of rows modified per minute greater than the `write-key-threshold` value.
+# Note:
+# * The default value of `write-key-threshold` is 0, which means that the feature does not split the table replication range according to the number of rows modified in a table by default.
+# * You can configure this parameter according to your cluster workload. For example, if it is configured as 30000, it means that the feature will split the replication range of a table when the number of modified rows per minute in the table exceeds 30000.
+# * When `region-threshold` and `write-key-threshold` are configured at the same time:
+#   TiCDC will check if the number of modified rows is greater than `write-key-threshold` first.
+#   If not, next check if the number of Regions is greater than `region-threshold`.
+write-key-threshold = 0
```

Comment:
In the Chinese version, it is 30000. See https://github.com/pingcap/docs-cn/pull/13578#discussion_r1193557054. Please help check and keep them consistent.